### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #1: Missing CONTRIBUTING.md file
Hey @eduardconstantin. Can I add the contributing.md file ? please assign me

### PR DESCRIPTION
**Pull request by lazyfuhrer**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Contributing to Crypto Rankings
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: please change the title

**Comment by lazyfuhrer** [comment]
@eduardconstantin check

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- [I Want To Contribute](#i-want-to-contribute)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting Enhancements](#suggesting-enhancements)
+- [Contributors](#contributors)
+
+
+
+## I Have a Question
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/eduardconstantin/Crypto-Rankings/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: add the correct link

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/eduardconstantin/Crypto-Rankings/issues/new).
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: same here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Reporting Bugs
+
+#### Before Submitting a Bug Report
+
+A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.
+
+- Make sure that you are using the latest version.
+- Determine if your bug is really a bug and not an error on your side e.g. using incompatible environment components/versions. If you are looking for support, you might want to check [this section](#i-have-a-question)).
+- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/eduardconstantin/Crypto-Rankings/issues?q=label:bug).
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: and here

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+- Possibly your input and the output
+- Can you reliably reproduce the issue? And can you also reproduce it with older versions?
+
+#### How Do I Submit a Good Bug Report?
+
+> You must never report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs must be sent by email to <>.
+
+We use GitHub issues to track bugs and errors. If you run into an issue with the project:
+
+- Open an [Issue](https://github.com/eduardconstantin/Crypto-Rankings/issues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: and here

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+
+## Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Crypto Rankings, **including completely new features and minor improvements to existing functionality**. Following these guidelines will help maintainers and the community to understand your suggestion and find related suggestions.
+
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version.
+- Perform a [search](https://github.com/eduardconstantin/Crypto-Rankings/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: and here

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider writing an add-on/plugin library.
+
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/eduardconstantin/Crypto-Rankings/issues).
```
Comment (eduardconstantin) [inline CONTRIBUTING.md]: and finally here

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by lazyfuhrer** [comment]
done please check one again @eduardconstantin

**Review by eduardconstantin — APPROVED** [review]

**Pull request merged**